app/cypress/.env.cypress

```diff
@@ -0,0 +1,63 @@
+# injected by heroku, hence without "ADALITE_" prefix
+PORT=3000
+REDIS_URL=''
+ADALITE_ENABLE_DEBUGGING=false
+ADALITE_SERVER_URL='https://localhost:3000'
+ADALITE_BLOCKCHAIN_EXPLORER_URL='https://explorer-testnet.adalite.io'
+ADALITE_DEFAULT_ADDRESS_COUNT=10
+ADALITE_GAP_LIMIT=20
+ADALITE_STATS_PWD=admin
+
+ADALITE_DEMO_WALLET_MNEMONIC='hill safe victory sun tired fetch police radio sport output buyer deny april fringe stumble'
+
+ADALITE_LOGOUT_AFTER=900
+ADALITE_SUPPORT_EMAIL='support@test.test'
+
+# to tell foreign requests from adalite ones
+ADALITE_BACKEND_TOKEN=""
+
+# to distinct between enviroments
+ADALITE_ENV='local'
+
+# mock server config
+ADALITE_ENABLE_SERVER_MOCKING_MODE=true
+ADALITE_MOCK_TX_SUBMISSION_SUCCESS=true
+ADALITE_MOCK_TX_SUMMARY_SUCCESS=true
+ADALITE_DEVEL_AUTO_LOGIN = false
+
+# for development of trezor firmware
+ADALITE_TREZOR_CONNECT_URL=
+
+# google analytics
+ADALITE_GA_TRACKING_ID=
+
+# mailchimp
+ADALITE_MAILCHIMP_API_KEY=
+ADALITE_MAILCHIMP_LIST_ID=
+
+# donations
+ADALITE_FIXED_DONATION_VALUE = 40
+ADALITE_MIN_DONATION_VALUE = 1
+
+# IP blacklist as comma separated list
+ADALITE_IP_BLACKLIST = 8.8.8.8, 9.9.9.9
+
+#sentry
+SENTRY_DSN='https://d77d3bf9d9364597badab9c00fa59a31@sentry.io/1501383'
```

Comment:
having the sentry dsn set for tests can mess up our error data

Reply:
@PeterBenc it is checked via SENTRY_DSN: check.nonEmptyString, is this necessary?

Reply:
I'm pretty sure it isn't - the sentry library itself should be graceful when no DSN is supplied and fall back to not doing anythin instead - you can try it yourself

Reply:
https://docs.sentry.io/product/sentry-basics/dsn-explainer/

> If an SDK is not initialized or if it is initialized with an empty DSN, the SDK will not send any data over the network, such as captured exceptions.